changelog/25269.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+ui: Fix copy button not working on masked input when value is not a string
```

Comment:
#25269 was reliant on changes made in #25235, so I just lumped them in together here so I wouldn't forget to open the backport for this after the JSON update backport was merged